### PR TITLE
feat/create project list client changes

### PR DESCRIPTION
**Pull request by markcsims**

### CODE REVIEW

File: apps/gp2-frontend/src/projects/__tests__/Projects.test.tsx [hunk 1]
```diff
@@ -1,11 +1,79 @@
...
+    <RecoilRoot
+      initializeState={({ set }) => {
+        set(refreshProjectsState, Math.random());
+      }}
+    >
+      <Suspense fallback="loading">
+        <Auth0Provider user={{}}>
+          <WhenReady>
+            <MemoryRouter initialEntries={['/projects']}>
+              <Route path="/projects">
```
Comment (jhonny-m) [inline apps/gp2-frontend/src/projects/__tests__/Projects.test.tsx:31]: shouldn't we use the routes?

File: packages/gp2-components/src/organisms/ProjectsBody.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+import { drawerQuery, pixels } from '@asap-hub/react-components';
+import { css } from '@emotion/react';
+import ProjectCard from './ProjectCard';
+
+const { perRem } = pixels;
+type ProjectsBodyProps = {
+  projects: gp2.ListProjectResponse;
+};
+const gridContainerStyles = css({
+  display: 'grid',
```
Comment (jhonny-m) [inline packages/gp2-components/src/organisms/ProjectsBody.tsx:11]: `display:'flex'`

File: packages/gp2-components/src/organisms/ProjectsBody.tsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
+import { gp2 } from '@asap-hub/model';
+import { drawerQuery, pixels } from '@asap-hub/react-components';
+import { css } from '@emotion/react';
+import ProjectCard from './ProjectCard';
+
+const { perRem } = pixels;
```
Comment (jhonny-m) [inline packages/gp2-components/src/organisms/ProjectsBody.tsx:6]: lets use the `rem` function instead

File: packages/gp2-components/src/organisms/ProjectsHeader.tsx [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+
+const props = {
+  image: projectsImage,
+  position: 'center',
+  title: 'Project Directory',
+  description:
+    'Explore past and present GP2 projects being carried out by teams within the network and discover open opportunities to join them.',
+};
+
+const ProjectsHeader: React.FC = () => <PageBanner {...props} />;
```
Comment (jhonny-m) [inline packages/gp2-components/src/organisms/ProjectsHeader.tsx:12]: I wonder whats the advantage of having this as a separate component

**Review by jhonny-m — APPROVED** [review]

File: packages/gp2-components/src/organisms/ProjectsBody.tsx (hunk 2 above)
Comment (markcsims) [reply packages/gp2-components/src/organisms/ProjectsBody.tsx:11]: Done

File: packages/gp2-components/src/organisms/ProjectsHeader.tsx (hunk 4 above)
Comment (markcsims) [reply packages/gp2-components/src/organisms/ProjectsHeader.tsx:12]: This is the same as working groups.

File: apps/gp2-frontend/src/projects/__tests__/Projects.test.tsx (hunk 1 above)
Comment (markcsims) [reply apps/gp2-frontend/src/projects/__tests__/Projects.test.tsx]: Done

**Pull request merged**